### ISSUE TITLE
When using LinearSampler in TextureFX SamplerState settings seem to be ignored

### ISSUE THREAD

**Issue by bj-rn** [vvvv/VL.Stride#382]
![image](https://user-images.githubusercontent.com/1632591/102798656-4de02000-43a9-11eb-89f1-31fb4fd8e6b2.png)
[SamplerStateDemo.zip](https://github.com/vvvv/VL.Stride/files/5725349/SamplerStateDemo.zip)

**Comment by tebjan** [comment]
yes, this pin doesn't make sense and will be removed, please don't use it at the moment. see also: https://github.com/vvvv/VL.Stride/issues/314

ideally, we will add a general "Sampler" input pin that will be used.

**Comment by bj-rn** [comment]
Don't use the Pin or don't use _LinearSampler_ in TextureFX?
Is it ok to use it if I need a sampler with "standard settings"?

**Comment by tebjan** [comment]
yes, if you want a fixed setting sampler, you can use any of the predefined ones in:
https://github.com/stride3d/stride/blob/9219cbbb9de7a238060d97cf1cfbbf759120aa23/sources/engine/Stride.Graphics/Shaders/Texturing.sdsl

we need to figure out how we implement a general input where one can connect a SamplerState node.

**Comment by bj-rn** [comment]
> yes, this pin doesn't make sense and will be removed, please don't use it at the moment. see also: #314
> ideally, we will add a general "Sampler" input pin that will be used.

The pin is still there as of  `2021.4.9 preview 971`. Is there going be a general "Sampler" pin?
